packages/components/src/custom-select-control-v2/styles.ts

```diff
@@ -109,6 +117,8 @@ export const SelectItem = styled( Ariakit.SelectItem )`
 	padding: ${ ITEM_PADDING };
 	font-size: ${ CONFIG.fontSize };
 	line-height: 2.15rem; // TODO: Remove this in default but keep for back-compat in legacy
+	user-select: none;
```

Comment:
| Before | After |
|---------|---------|
| ![Screen Recording on 2024-06-07 at 01-46-27](https://github.com/WordPress/gutenberg/assets/81248/50096ad7-dd60-4420-bbb9-9f68bac8ae62)  | ![Screen Recording on 2024-06-07 at 01-45-24](https://github.com/WordPress/gutenberg/assets/81248/7cf06b3a-8e17-4785-ab99-a6906287bb7f) |

Reply:
I don't personally see this as an issue that needs fixing — neither Ariakit, nor the V1 version of the component disallowed user selection on the individual option items.

Reply:
On second thoughts, I double checked what other libraries do and there seems to be alignment in adding `user-select: none`. I will add it to https://github.com/WordPress/gutenberg/pull/62825